web/src/message_list.ts

```diff
@@ -216,6 +215,7 @@ export class MessageList {
 
         if (interior_messages.length > 0) {
             this.view.rerender_preserving_scrolltop(true);
+            this.update_user_sidebar_participants();
```

Comment:
Why is this here?

Reply:
Since `this.view.rerender_preserving_scrolltop(true);` doesn't update participants, we need to update them here.

Reply:
OK, but why is it in the `interior_messages` block, and also further down in this same function?

Reply:
Oh, it's because we return. Right.